server.R

```diff
@@ -373,7 +403,8 @@ mnwd_baseline <- function(){
                             paste("BR", 1:num_tiers, sep=""),
                             "baseline_variable_bill")
   bill_info$baseline_bill <- bill_info$baseline_variable_bill + 11.39
-  
+  #adding baseline usage
+  bill_info$baseline_usage <- bill_info$B1 + bill_info$B2 + bill_info$B3 + bill_info$B4 + bill_info$B5
```

Comment:
Should probably do something like `select(matches("[B][0-9]")) %>% rowSums()`  like you did before, so that we can account for changing numbers of tiers.